memory_engine/memory_store.py

```
@@ -343,12 +345,12 @@ def get_stats(self) -> Dict:
-        return {
-            "total_events": len(self.memory),
-            "critical_events": sum(1 for e in self.memory if e.is_critical),
-            "avg_age_hours": np.mean(ages) if np is not None else sum(ages) / len(ages) if ages else 0,
-            "max_recurrence": max(e.recurrence_count for e in self.memory),
-        }
+    return {
+        "total_events": len(self.memory),
+        "critical_events": sum(1 for e in self.memory if e.is_critical),
+        "avg_age_hours": np.mean(ages) if np is not None else sum(ages) / len(ages),
+        "max_recurrence": max(e.recurrence_count for e in self.memory),
+    }
```

Ruff (0.14.11)

`return` statement outside of a function/method

(F706)